src/Microsoft.ML.LightGbm/WrappedLightGbmInterface.cs

```diff
@@ -225,6 +226,55 @@ public static string JoinParameters(Dictionary<string, object> parameters)
             return string.Join(" ", res);
         }
 
+        // Static override name map that maps friendly names to lightGBM arguments.
+        // If an argument is not here, then its name is identicaltto a lightGBM argument
+        // and does not require a mapping, for example, Subsample.
+        private static Dictionary<string, string> _nameMapping = new Dictionary<string, string>()
```

Comment:
I dont like having a dictionary for this, even worse is having the dictionary defined here in this interface. If someone were to add new options to be handled in lightgbm, its not clear that they need to potentially update this as well in order for the correct lightgbm argument to get mapped. I think a better "option" (har har no pun intended) would be to have the Option classes build this dictionary, but I haven't worked on this yet and this at least gets things semi-working.